test/LaunchDarkly.EventSource.Tests/EventSourceReconnectingTest.cs

```diff
@@ -82,7 +82,7 @@ public void RetryDelayDurationsShouldIncrease()
             {
                 handler.QueueResponse(StubResponse.StartStream(
                     StreamAction.Write(":hi\n"),
-                    StreamAction.CloseStreamAbnormally()));
```

Comment:
I'm not sure what I intended the difference between CloseStream and CloseStreamAbnormally to be, but it didn't really contribute anything to this test.